### DISCUSSION TITLE
Exemple architecture full-stack project

### DISCUSSION THREAD

**Discussion by SavinDevelop** [litestar-org/litestar#3668] [General]
Hello, I want to create a project based on litestar, so I would like to get recommendations and examples of project architects based on this framework. I am especially interested in full-stack with htmx

**Comment by JacobCoffee** [comment]
Here are some if it helps get started:

- https://github.com/litestar-org/litestar-fullstack
- https://github.com/litestar-org/litestar-litestack/ (when it becomes public)
- https://github.com/JacobCoffee/niapi
- https://github.com/euri10/litestar_saq_htmx

**Reply by SavinDevelop** [reply]
Thanks!

**Reply by euri10** [reply]
you can also see pretty much all htmx examples used with Litestar here: https://gitlab.com/euri10/htmx-examples-litestar
